manim/mobject/mobject.py

```diff
@@ -66,6 +77,13 @@ class Mobject(Container):
 
     """
 
+    animation_overrides: "Dict[Type[Animation], List[Dict[Type[Mobject], str]]]" = {}
```

Comment:
```suggestion
    animation_overrides: Dict[Type[Animation], List[Dict[Type[Mobject], str]]] = {}
```

Reply:
Is there a workaround for Mobject as well? I cannot use it inside the class... I changed it to `Dict[Type[Animation], List[Dict[Type["Mobject"], str]]]`